src/CircuitBreaker/src/Hystrix/Collapser/RequestBatch.cs

```diff
@@ -74,7 +74,7 @@ public int Size
                     collapsedRequest.CompletionSource = tcs;
 
                     CollapsedRequest<TRequestResponse, TRequestArgument> existing =
-                        arg == null ? GetOrAddNullArg(collapsedRequest) : _argumentMap.GetOrAdd(arg, collapsedRequest);
+                        arg is null ? GetOrAddNullArg(collapsedRequest) : _argumentMap.GetOrAdd(arg, collapsedRequest);
```

Comment:
@dtillman `arg` is of unconstrained generic type `TRequestArgument`, so comparing to `null` doesn't make sense when it's a value type (resulting in S2955 warning). How do you think this should be solved? I've tried adding a `class` constraint, but that results in errors.